runner/hooks/started.d/job-start

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Probably this file might better be named `job_started.d/update_status`, so that we won't over complete this script in the future(I'd like to keep one hook script do only one thing, which might be better enforced by naming this script `update_status` rather than a more general `job-start`.

Reply:
I've renamed all hook files for consistency to use job prefix and dashes

```
runner@k8s:/$ find /etc/arc/
/etc/arc/
/etc/arc/hooks
/etc/arc/hooks/job-completed.d
/etc/arc/hooks/job-completed.d/update-status
/etc/arc/hooks/job-started.d
/etc/arc/hooks/job-started.d/update-status
/etc/arc/hooks/job-started.sh
/etc/arc/hooks/job-completed.sh

```